src/app/contextBroker/contextBroker.cpp

```diff
@@ -380,6 +384,7 @@ PaArgument paArgs[] =
   { "-writeConcern",  &writeConcern, "WRITE_CONCERN",  PaInt,    PaOpt, 1,          0,      1,     WRITE_CONCERN_DESC },
 
   { "-corsOrigin",       allowedOrigin,     "ALLOWED_ORIGIN",    PaString, PaOpt, _i "",          PaNL,  PaNL,     ALLOWED_ORIGIN_DESC    },
+  { "-enableCORS",       corsEnabled,       "CORS_ENABLED",      PaString, PaOpt, _i "",          PaNL,  PaNL,     CORS_ENABLED_DESC      },
```

Comment:
Why not simply `-cors` ... ?
'enable' is kind of there implicitly when a CLI option is used.